pkgs/development/libraries/dbus/make-dbus-conf.nix

```diff
@@ -1,6 +1,7 @@
 { runCommand, writeText, libxslt, dbus
 , serviceDirectories ? []
 , suidHelper ? "/var/setuid-wrappers/dbus-daemon-launch-helper"
+, apparmor ? "enabled" # one of enabled, disabled, required
```

Comment:
if this is changed https://github.com/NixOS/nixpkgs/pull/102537/files#r519515886 it should be here as well